src/components/Plans/Checkout/PurchaseSummary/PurchaseSummary.js

```diff
@@ -489,7 +489,7 @@ export const PurchaseSummary = InjectAppServices(
         case PLAN_TYPE.byEmail:
           return paymentMethodType === paymentType.creditCard ? (
             <div>
-              {!isFree && currentMonthTotal === 0 && (
+              {!isFree && majorThat21st === true && (
```

Comment:
Si es un boolean, bastaría con comprobarlo de esta forma:
```suggestion
{!isFree && majorThat21st && (
```